loopy/target/pyopencl_execution.py

```diff
@@ -274,6 +274,9 @@ def get_invoker_uncached(self, kernel, codegen_result):
         generator = PyOpenCLExecutionWrapperGenerator()
         return generator(kernel, codegen_result)
 
+    def get_wrapper_generator(self):
+        return PyOpenCLExecutionWrapperGenerator()
```

Comment:
This functionality shouldn't be PyOpenCL-specific (i.e. if you'd like this added, it should be available on every target).

Reply:
Good point, I'll update the PR to make this more generic.

Reply:
Done.